### PR TITLE
accelerator/cuda: fix typos

### PR DESCRIPTION
**Pull request by wenduwan**
Fix a couple minor typos to make compiler happy

### CODE REVIEW

**Comment by wenduwan** [comment]
Running AWS CI

**Review by bosilca — APPROVED** [review]

**Comment by wenduwan** [comment]
Had to made a couple more adjustments. Requesting review from @devreal

File: opal/mca/accelerator/cuda/accelerator_cuda.c [hunk 1]
```diff
@@ -498,13 +498,14 @@ static int accelerator_cuda_memcpy(int dest_dev_id, int src_dev_id, void *dest,
      * Additionally, cuMemcpy is not necessarily always synchronous. See:
      * https://docs.nvidia.com/cuda/cuda-driver-api/api-sync-behavior.html
      * TODO: Add optimizations for type field */
-    result = cuMemcpyAsync((CUdeviceptr) dest, (CUdeviceptr) src, size, GET_STREAM(opal_accelerator_cuda_memcpy_stream.super));
+    result = cuMemcpyAsync((CUdeviceptr) dest, (CUdeviceptr) src, size,
+                           GET_STREAM(&opal_accelerator_cuda_memcpy_stream.base));
```
Comment (wenduwan) [inline opal/mca/accelerator/cuda/accelerator_cuda.c:502]: @devreal Does this change do what you intended it for?

Comment (edgargabriel) [reply opal/mca/accelerator/cuda/accelerator_cuda.c:502]: why do you need the GET_STREAM macro in this particular scenario?  I think you can just use the opal_accelerator_cuda_memcpy_stream.stream here, there is no ambiguity with the default stream (which is what the GET_STREAM macro was designed to do), isn't it?

Comment (wenduwan) [reply opal/mca/accelerator/cuda/accelerator_cuda.c]: @edgargabriel I think you are right. Updated the PR.

**Review by edgargabriel — APPROVED** [review]
LGTM

**Comment by wenduwan** [comment]
Merging the change for now. This is currently blocking CI on main branch.

**Pull request merged**